### PR TITLE
NWSS: post-staging fixes/updates

### PR DESCRIPTION
**Pull request by dsweber2**
The first commit is some changes that George recommended after the original PR was merged. This will also be a branch for any changes that need to be made to handle problems found during staging.

### CODE REVIEW

File: ansible/vars.yaml [hunk 1]
```diff
@@ -53,6 +53,9 @@ doctor_visits_midas_password: "{{ vault_doctor_visits_midas_password }}"
 # NCHS
 nchs_mortality_token: "{{ vault_nchs_mortality_token }}"
 
+# NWSS
+nwss_mortality_token: "{{ vault_nchs_mortality_token }}"
```
Comment (melange396) [inline ansible/vars.yaml:57]: maybe we can eventually change `vault_nchs_mortality_token` to `vault_socrata_token` or perhaps `vault_cdc_socrata_token`, but thatll need help from someone who knows how to do the vault stuff...  @minhkhul is that difficult?

Comment (minhkhul) [reply ansible/vars.yaml:57]: Nope not difficult. Will do that change after   this indicator ships though, since it involve another repo.

Comment (minhkhul) [reply ansible/vars.yaml:57]: I was wrong! The secrets _are_ in this repo. Will make changes shortly.

Comment (minhkhul) [reply ansible/vars.yaml]: Changes are ready for review.

Comment (melange396) [reply ansible/vars.yaml]: i (perhaps obviously) dont know whats going on with the `vault.yaml` changes, but the rest of it looks good to me!

**Comment by melange396** [comment]
its not directly related to the scope of this venture, but its close enough in form to include here as well: itd be good if we changed `token` to `socrata_token` in nchs_mortality the way we did for nwss_wastewater.  if nobody else volunteers to make the edits, i can do them after im done with a few other things.

**Comment by dsweber2** [comment]
done; Minh had already covered like half, this was just the remaining

**Comment by melange396** [comment]
awesome, thanks!

File: nchs_mortality/tests/test_pull.py [hunk 2]
```diff
@@ -8,70 +8,91 @@
...
-            nyc_list = raw_df.loc[(raw_df["state"] == "New York City")
-                                & (raw_df[metric].isnull()), "timestamp"].values
-            final_list = df.loc[(df["geo_id"] == "ny")
-                                & (df[metric].isnull()), "timestamp"].values
+            ny_list = raw_df.loc[
+                (raw_df["state"] == "New York") & (raw_df[metric].isnull()), "timestamp"
+            ].values
+            nyc_list = raw_df.loc[
+                (raw_df["state"] == "New York City") & (raw_df[metric].isnull()),
+                "timestamp",
```
Comment (minhkhul) [inline nchs_mortality/tests/test_pull.py:65]: Should trailing comma at end of list be consistent (i.e. either always end in a comma like line 65 or end with no comma like line 61)?
```suggestion
                "timestamp"
```

Comment (dsweber2) [reply nchs_mortality/tests/test_pull.py:65]: oh whoops. My styler was unhappy with this file, and I accidentally included it's changes.

**Comment by minhkhul** [comment]
@dsweber2 How much of the changes in nwss_wastewater in this PR (that went stale because of negative values from my understanding) is still relevant? These ansible and nchs_mortality changes should be going out at some point after a small vault update. But I'm not sure how much of the nwss_wastewater part still holds now.

**Comment by dsweber2** [comment]
it's mostly name changes and docs which still apply. I probably should've marked this as non-draft and based my other PR on it

**Review by minhkhul — APPROVED** [review]
Note: vault change is already in main.

**Pull request merged**